ATC/src/mockRadar.h

```diff
@@ -0,0 +1,15 @@
+#include <map>
```

Comment:
Same thing, needs a header guard (even though we'll remove the file eventually once the actual radar is in place)

Reply:
Added in b1d9fb7e3deffaf386c7a3ce2963011804976860